commander/src/bootstrapping/templates/lisk-template-ts/templates/command/src/app/modules/commands/command.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable class-methods-use-this */
+
+import {
+    BaseCommand,
+    CommandVerifyContext,
+    CommandExecuteContext,
+	VerificationResult,
```

Comment:
The following imports are un-used.
`
CommandVerifyContext,
CommandExecuteContext,
VerificationResult,
`